testdata/baselines/reference/submodule/conformance/partiallyNamedTuples2.types.diff

```diff
@@ -0,0 +1,12 @@
+--- old.partiallyNamedTuples2.types
++++ new.partiallyNamedTuples2.types
+@@= skipped -37, +37 lines =@@
+ const matches = x.get(id1);
+ >matches : Iterable<[id2: string, object]>
+ >x.get(id1) : Iterable<[id2: string, object]>
+->x.get : <Key extends [id1: string, id2: string] | [id1: string] | []>(...key: Key) => GetResult<[id1: string, id2: string], Key, object>
++>x.get : <Key extends [] | [id1: string] | [id1: string, id2: string]>(...key: Key) => GetResult<[id1: string, id2: string], Key, object>
```

Comment:
For some reason the ordering I have in the submodule does not match for this one case. I'll have to recheck it for mistakes.

Reply:
I found one bug, but it was benign. Not sure where this difference comes from.